Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,178 @@
+            self.basic = HTTPBasicAuth(username, password)
+            self.host = host
+
+        def __call__(self, r):
+            if r.url.startswith(self.host):
+                return self.basic(r)
+            else:
+                return r
+
+    serverIp=demisto.params()['server-ip']
+    username=demisto.params()['credentials']['identifier']
+    password=demisto.params()['credentials']['password']
+    savedReportId = demisto.params()['saved-Report-id']
+
+
+    wsdl = 'https://{}/ProtectManager/services/v2011/incidents?wsdl'.format(serverIp)
+
+    session = Session()
+    session.auth = SymantecAuth(username,password, "https://{}".format(serverIp))
+    session.verify = False
+    transport = Transport(session=session)
+
+    client = Client(wsdl=wsdl, transport=transport)
+
+    def  getIncidentId(dtobj):
+
+        if type(dtobj) is datetime.datetime:
+            return client.service.incidentList(savedReportId=savedReportId, incidentCreationDateLaterThan=dtobj)
+        else:
+            raise ValueError('excpected datetime object')
+
+
+
+    def incidentDetail(_id):
+
+        return client.service.incidentDetail(incidentId=_id,includeViolations=True,includeHistory=True)
+
+
+
+    def fetchincidents():
+        last_run=demisto.getLastRun()
+        if 'lt_time' not in last_run:
+            last_run['lt_time'] = datetime.datetime.strptime("01/11/2018 00:00:00.000000",'%d/%m/%Y %H:%M:%S.%f')
+        else:
+            last_run['lt_time']= datetime.datetime.strptime(last_run['lt_time'],'%d/%m/%Y %H:%M:%S.%f')
+
+        incidentids=getIncidentId(last_run['lt_time'])
+        incidents=[]
+        if len(incidentids['incidentId']) > 0:
+            for incident1 in incidentids['incidentId']:
+                inc=incidentDetail(str(incident1))
+                incidents.append({
+                    'name':"DLP Incident #"+str(incident1),
+                    'details':  inc[0]['incident']['violatedPolicyRule'][0]['ruleName']+ "RULE has been violated"
+                })
+                if inc[0]['incident']['incidentCreationDate'].isoformat() > last_run['lt_time'].isoformat():
+                    demisto.setLastRun({'lt_time':inc[0]['incident']['incidentCreationDate'].strftime("%d/%m/%Y %H:%M:%S.%f")})
+        demisto.incidents(incidents)
+        return(incidents)
+
+
+    def updateIncidents():
+        args = demisto.args()
+        inc_id = args['incidentId']
+        args.pop('incidentId')
+        if args['status'] == 'New':
+            args['status'] = 'incident.status.New'
+        return client.service.updateIncidents(updateBatch={'batchId':str(random.randint(11111,99999)),'incidentId':inc_id,'incidentAttributes':args})
+
+
+
+    def myconverter(o):
+        if isinstance(o, datetime.datetime):
+            return o.__str__()
+
+
+
+    if demisto.command() == 'fetch-incidents':
+        fetchincidents()
+    elif demisto.command() =='test-module':
+        demisto.results("ok")
```

Comment:
Add meaningful test here. Currently the test always will be successful

Reply:
added a test to pull a list of incidents